adapters/MythicMobsAdapter/src/main/kotlin/com/caleb/typewriter/mythicmobs/entries/event/MythicMobDeathEventEntry.kt

```diff
@@ -0,0 +1,33 @@
+package com.caleb.typewriter.mythicmobs.entries.event
+
+import io.lumine.mythic.bukkit.events.MythicMobDeathEvent
+import lirand.api.extensions.server.server
+import me.gabber235.typewriter.adapters.Colors
+import me.gabber235.typewriter.adapters.Entry
+import me.gabber235.typewriter.adapters.modifiers.Help
+import me.gabber235.typewriter.entry.*
+import me.gabber235.typewriter.entry.entries.EventEntry
+import me.gabber235.typewriter.utils.Icons
+import java.util.*
+
+
+@Entry("on_mob_die", "When a player kill a MythicMobs mob.", Colors.YELLOW, Icons.SQUARE_CHECK)
+class MythicMobDeathEventEntry(
+	override val id: String = "",
+	override val name: String = "",
+	override val triggers: List<String> = emptyList(),
+	@Help("The mob's name use to check.")
```

Comment:
Make this more descriptive, like `Only trigger when a specific mob dies`. Also you could make this field optional to allow users to trigger something if any mythic mob dies.